oc-chef-pedant/spec/shared_context/keys_context.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright: Copyright (c) 2015 Chef Software, Inc.
+# License: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def rand_id
+  rand(10**7...10**8).to_s
+end
+
+RSpec.shared_context 'when the current_requestor is the superuser' do
+  let(:current_requestor) { superuser }
+end
+
+RSpec.shared_context 'when the current_requestor is a user in the main org' do
+  let(:test_user_name) { "pedant_test_user_#{rand_id}" }
```

Comment:
Let's have the name indicate the context `"pedant_test_user_keys_#{rand_id}"`. Makes it easier to identify the source when we sometimes  find that we're not cleaning up properly.